src/components/text/LargeParagraph.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const LargeParagraph = ({ text }) => 
+	<main className="pa3 pa5-ns">
```

Comment:
Should we leave these as `<main>` tags or change them to `<section>` or `<div>` tags? Just thinking about semantics.